### ISSUE TITLE
First search for an item shows double the amount of auctions per seller

### ISSUE THREAD

**Issue by raethkcj** [shirsig/aux-addon#366]
This only seems to happen for "commodity" items, like cloth, that there are many pages of.

Example, my first search for Runecloth:
![image](https://github.com/shirsig/aux-addon/assets/7716908/347c5fdd-dfc3-4549-9972-e854b2714265)
And then pressing Search again immediately after the search is finished:
![image](https://github.com/shirsig/aux-addon/assets/7716908/bd252a7e-21f4-4482-8693-cf88d84c283f)

aux 3.2.5 in Wrath Classic 3.4.2

**Comment by shirsig** [comment]
I was not able to reproduce this. Could it be a conflict with another (probably auction related) addon?

**Comment by raethkcj** [comment]
I was able to reproduce it just now with v3.2.6, again in Wrath with Runecloth.

As far as I can tell, it's only happening for the first search of an item per game session. If I reload, or log out to character select and back in, searching for Runecloth returns correct results the first time. A full client restart is required to reproduce it for the same item again.

Interestingly, I can actually see the doubling happening as the results are coming in:
https://github.com/shirsig/aux-addon/assets/7716908/c9386c64-ddd2-480d-b639-b4042cfe6026

**Comment by shirsig** [comment]
I still cannot reproduce it. Are you sure it's not a conflict with another addon?

**Comment by raethkcj** [comment]
Yes, this is with zero addons enabled besides Aux. Sorry, I meant to say that in the last comment but I see I forgot.
